modules/programs/mbsync-accounts.nix

```diff
@@ -53,5 +53,27 @@ with lib;
         Pattern of mailboxes to synchronize.
       '';
     };
+
+    extraConfig.store = mkOption {
```

Comment:
Maybe `extraConfig.local` is a clearer name? It would also match the `offlineimap` naming. Could also add an `extraConfig.remote`…